docs/unified/msi/faq.md

```diff
@@ -71,37 +73,37 @@ Please follow the [recovery procedure](recovery.md).
 
 ## Which CPUs are supported?
 
-We can say the "supported" one is the one that have been tested during
+We can say the "supported" one is the one that has been tested during
 Validation Procedure or have been tested by the community.
 
 The list of all supported CPUs is available in the [CPU HCL](hcl.md)
 documentation.
 
 In practice, any Alder Lake-S Processor should work.
 
-12900KS is rather unknown at the moment, because it was released later and may
-require more recent microcode.
+12900KS is rather unknown at the moment because it was released later and may
+require a more recent microcode.
```

Comment:
This is also outdated. NOw we wait for 14900KS actually, which is not out yet

Reply:
Fixed: bc98718eb918